pkg/config/generate.go

```diff
@@ -477,5 +484,11 @@ func (g *simpleGenerator) Generate(cs *api.OpenShiftManagedCluster) (err error)
 		c.ServiceCatalogClusterID = uuid.NewV4()
 	}
 
+	// TODO: Make this better configurable when clear how
+	if len(c.RHUsername) == 0 {
+		c.RHUsername = os.Getenv("RH_USERNAME")
```

Comment:
We have moved envs out of the plugin code, let's keep it that way.

Reply:
This is temporary. We need to define how we want to manage there overall...

Reply:
Even if temporary, I think it's still preferrable to consolidate with the rest of the envs